browser/ui/content_settings/brave_autoplay_blocked_image_model.cc

```diff
@@ -18,32 +18,27 @@ BraveAutoplayBlockedImageModel::BraveAutoplayBlockedImageModel()
     : ContentSettingSimpleImageModel(ImageType::PLUGINS,
                                      CONTENT_SETTINGS_TYPE_AUTOPLAY) {}
 
-void BraveAutoplayBlockedImageModel::UpdateFromWebContents(
-    WebContents* web_contents) {
-  set_visible(false);
+bool BraveAutoplayBlockedImageModel::UpdateAndGetVisibility(WebContents* web_contents) {
   if (!web_contents)
-    return;
+    return false;
```

Comment:
There is no longer a `set_visible(bool)` and instead this call returns if it should be visible or not.